scripts.py

```diff
@@ -1312,9 +1312,9 @@ def arg_parser(cls, _db):
         parser.add_argument(
             '--default-medium-type',
             help=u'Default medium type used in the case when it\'s not explicitly specified in a metadata file. '
-                 u'Valid values are: Book, Audio.',
+                 u'Valid values are: EditionConstants.FULFILLABLE_MEDIA.',
```

Comment:
I think you changed the default_medium_type but not this piece, which is the confusing part.

Reply:
That's weird, I must've forgotten to push